_rules/SC3-1-1-html-xml-lang-match.md

```diff
@@ -1,5 +1,5 @@
 ---
-name: SC3-1-1-html-xml-lang-match
+name: Matching HTML lang and xml:lang attribute
```

Comment:
Why is this here?

I would suggest "HTML lang and xml:lang attribute matches"

Reply:
Ditto. In general, can we please refrain from introducing unrelated changes in pull requests?